Jenkinsfile_CNP

```diff
@@ -0,0 +1,29 @@
+#!groovy
+
+@Library("Infrastructure")
+
+def type = "nodejs"
+def product = "em"
+def component = "icp-api"
```

Comment:
Should the component name just be - "icp" ? Which will result in the chart name as "em-icp"